keras_rcnn/layers/pooling.py

```diff
@@ -55,3 +57,48 @@ def call(self, x, **kwargs):
 
     def compute_output_shape(self, input_shape):
         return None, self.regions, self.size, self.size, self.channels
+
+
+class ROIAlign(ROI):
+    """ROIAlign pooling layer proposed in Mask R-CNN (Kaiming He et. al.).
+
+    # Parameters
+    size: Fixed size [h, w], e.g. [7, 7], for the output slices.
+    regions: Integer, number of regions of interest.
+    stride: Integer, pooling stride.
+
+    # Returns
+    4D Tensor (number of regions, slice_height, slice_width, channels)
+    """
+
+    def __init__(self, size, regions, stride=1, **kwargs):
+        self.stride = stride
+        super(ROIAlign, self).__init__(size=size,
+                                       regions=regions,
+                                       **kwargs)
+
+    def call(self, x, **kwargs):
+        image, regions = x[0], x[1]
+
+        # convert regions from (x, y, w, h) to (x1, y1, x2, y2)
+        regions = keras.backend.cast(regions, keras.backend.floatx())
+        regions = regions / (self.stride + 0.0)
```

Comment:
@0x00b1 No basically what happens here is to cast the stride to a float. So probably `keras.backend.cast(self.stride, keras.backend.floatx())` is better.